english/computer-window/computer-window.md

```diff
@@ -7,3 +7,7 @@ You asked her,
  And she says,
 
  - Yes, thank you. I'm just thinking about our future...
+
+ What do you do?
+
+ (Check out Wikipedia)[../wikipedia/wiki.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.